bin/varnishd/cache/cache_tcp_pool.c

```diff
@@ -240,8 +247,26 @@ VCP_New(struct conn_pool *cp, const void *id, void *priv,
 	VTAILQ_INIT(&cp->connlist);
 	VTAILQ_INIT(&cp->killlist);
 
+	// XXX layering
+	CHECK_OBJ_NOTNULL(tp, TCP_POOL_MAGIC);
+	// XXX common cluster/seg
+	if (tp->uds != NULL)
+		cp->stats = VSC_vcp_New(NULL, &cp->vsc, "%p.%s", id, tp->uds);
+	else {
+		if (! tp->ip4 || (cache_param->prefer_ipv6 && tp->ip6))
+			addr = tp->ip6;
+		else
+			addr = tp->ip4;
+
+		AN(addr);
+		VTCP_name(addr, abuf, sizeof abuf, pbuf, sizeof pbuf);
+		cp->stats = VSC_vcp_New(NULL, &cp->vsc, "%p.%s:%s", id,
+					abuf, pbuf);
```

Comment:
Until Varnish 4.0 backend counters used to include the inet4/inet6/port tuple in the name, for example:

```
VBE.default(127.0.0.1,::1,8080).*
```

I think we should inherit that behavior for the new counters.